gstd/src/exec.rs

```diff
@@ -0,0 +1,19 @@
+// This file is part of Gear.
```

Comment:
Useless file

Reply:
Why?

Reply:
Ah, I see. We can just reimport `gcore::exec` indeed.

Reply:
Fixed.